mlibrary/catalog-browse
# Updating dependencies and removing chokidar-cli

# Overview
Updating `npm` dependencies for the sake of security.

This needs to be a monthly task.

## chokidar-cli
[`chokidar-cli`](https://www.npmjs.com/package/chokidar-cli) had several vulnerabilities, and has not been updated in 8 months. There is an [issue](https://github.com/open-cli-tools/chokidar-cli/issues/105) that has been open for a few months to fix it. The package is only used to watch CSS changes, which is nice but is not required for development. I have made the decision to remove it.

## Testing
* Remove and install packages (`rm -rf node_modules/ && docker-compose run --rm web npm install`).
* Launch the [site](http://localhost:4567/callnumber?query=UM1) and see if everything looks good.
* Trying and build the css (`docker-compose run --rm web npm run build`).
